artifacthub/library/general/allowedrepos/1.0.0/artifacthub-pkg.yml

```diff
@@ -0,0 +1,22 @@
+version: 1.0.0
+name: k8sallowedrepos
+displayName: Allowed Repositories
+createdAt: "2022-09-16T16:22:28Z"
```

Comment:
looks like this gets updated on every generate, i am not sure if we want that. can we have a create and modified field that only gets updated if the rego is updated?

Reply:
Updated the logic to only add this when first creating the metadata file.